kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Product.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.util.Objects;
+
+public class Product {
+    private final String name;
+    private final double price;
```

Comment:
In real application, when representing prices, you should use BigDecimal - but here it can stay as it is